x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -318,6 +319,25 @@ export function AlertDetails() {
       'data-test-subj': 'metadataTab',
       content: metadataTab,
     },
+    {
+      id: 'investigationguide',
+      name: (
+        <FormattedMessage
+          id="xpack.observability.alertDetails.tab.investigationGuideLabel"
+          defaultMessage="Investigation guide"
+        />
+      ),
+      'data-test-subj': 'investigationGuideTab',
+      disabled: !rule?.artifacts?.investigation_guide?.blob,
+      content: (
+        <>
+          <EuiSpacer size="m" />
+          <EuiMarkdownFormat>
+            {(rule?.artifacts?.investigation_guide?.blob as string) ?? ''}
```

Comment:
With the nullish coalescer, I don't think we need the pesky `as` here. 

<img width="478" alt="Screenshot 2025-05-23 at 1 13 35 PM" src="https://github.com/user-attachments/assets/bbbbeff5-6590-4f89-a87e-37393f974995" />
<img width="480" alt="Screenshot 2025-05-23 at 1 13 44 PM" src="https://github.com/user-attachments/assets/584418f0-dc1d-459d-9364-845ce5ad6ed5" />